pcdsdevices/mpod_apalis.py

```diff
@@ -120,11 +124,38 @@ def set_current(self, current_number: float) -> None:
 
     @max_voltage.sub_value
     def _new_max_voltage(self, value: float, **kwargs) -> None:
-        """Set explicit limits on voltage for better error reporting."""
-        bounds = (0, value)
+        self._update_max_voltage_limits(value or 1)
+
+    def _update_max_voltage_limits(self, value: Optional[float] = None) -> None:
+        """
+        Set explicit symmetric limits on voltage for better error reporting.
+
+        Refers to the parent module if it exists, to determine the limit scaling
+
+        If not, defaults to setting limits to positive definite
+        """
+        # Hacky, we need to be careful not to call this without a value from
+        # inside a callback, probably
+        if value is None:
+            value = self.max_voltage.get()
+
+        polarity = getattr(self, "biological_parent.polarity", Polarity.POSITIVE)
+        limit_scales = getattr(self, "biological_parent.limit_scales", (0, 100))
```

Comment:
Properties are usually safe to reference in a callback, but in this case the properties might get from epics io, so it could cause a callback threading timeout issue.